drivers/hud_twenty_twenty_two_to_twenty_twenty_four/app/concerns/hud_twenty_twenty_two_to_twenty_twenty_four/references.rb

```diff
@@ -0,0 +1,34 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudTwentyTwentyTwoToTwentyTwentyFour
+  module References
```

Comment:
Should this be something like CsvReferences?  I think you mentioned you'd need DB references as well, but this one reads a CSV file, which feels like it's probably not reusable for the db.

Reply:
Oh, maybe that's the model vs file below.